### PR TITLE
Added controller, model, repository for drop down questions

### PR DESCRIPTION
**Pull request by hassanidea**

### CODE REVIEW

File: src/main/java/com/SurveyMonkey/model/DropDownQuestion.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package com.SurveyMonkey.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class DropDownQuestion {
```
Comment (keving3ng) [inline src/main/java/com/SurveyMonkey/model/DropDownQuestion.java:9]: should extended the qtype abstract class with the question changes i showed taher yesterday, once we get those changes into main you can merge main branch and fix this class

File: src/main/java/com/SurveyMonkey/model/CreateSurvey.java [hunk 2]
```diff
@@ -30,31 +33,40 @@ public void setId(Long id) {
...
-//
-//    public void setNumberOfQuestions(int numberOfQuestions) {
-//        this.numberOfQuestions = numberOfQuestions;
-//    }
 
     public List<MultipleChoiceQuestion> getMultipleChoiceQuestions() {
         return multipleChoiceQuestions;
     }
 
+    public List<DropDownQuestion> getDropDownQuestionsQuestions() {
```
Comment (keving3ng) [inline src/main/java/com/SurveyMonkey/model/CreateSurvey.java:41]: should be just one list, but can wait for other changes to hit main

**Review by keving3ng — COMMENTED** [review]
have to merge in the abstract class for questions changes and then update your classes before merging

File: src/main/java/com/SurveyMonkey/model/DropDownQuestion.java (hunk 1 above)
Comment (TaherShabaan99) [reply src/main/java/com/SurveyMonkey/model/DropDownQuestion.java:9]: I thought Questions.java would be the abstract class and the other classes that you created in milestone one wouldn't be used as we already created 3 classes that are related to the question types.

**Pull request closed without merging**